sema/commons/j2/j2_functions.py

```
@@ -24,20 +25,25 @@ def all():
     @staticmethod
-    def all():
+    def all() -> dict:
         return {
             "xsd": xsd_format,
             "uri": uri_format,
         }
 
 
-def xsd_value(content, quote, type_name, suffix=None):
+def xsd_value(
+    content: any, quote: str, type_name: str, suffix: str | None = None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)